core/config.go

```diff
@@ -20,3 +27,38 @@ func TestConfig(t *testing.T) *Config {
 	})
 	return cfg
 }
+
+// LoadConfig config from the file under the given 'path'.
+// NOTE: Unfortunately, Core(Tendermint) does not provide us with convenient function to load the Config,
+// so we have to do our own. Furthermore, we have to make own SaveConfig, as the one the Core provides has
+// incompatibility issues with our toml parser version.
```

Comment:
Please us the same toml version as tendermint master, or, help them update their toml version.

Reply:
Seems to already be the case btw: https://github.com/tendermint/tendermint/blob/fb9eaf576afbcfcd83d265d7cc0f7d55f2e59393/go.mod#L6

Reply:
Ok, thanks. Resolved?